### PR TITLE
11999 transifex setup

### PR DESCRIPTION
**Pull request by smoyte**
Translations needed to be tidied because Transifex was choking on some duplicate keys. They were in need of a tidy anyway.

### CODE REVIEW

**Comment by smoyte** [comment]
@DevneyHamilton I have some spec failures to fix here. Please stay tuned...

**Comment by smoyte** [comment]
@DevneyHamilton ok ready!

File: README.md [hunk 1]
```diff
@@ -42,6 +42,30 @@ mailcatcher
...
+
+This combines all the *.en.yml files in the project into one temporary file and uploads them.
+We do it this way to make the translator's job easier as they only have to translate one resource.
+
+To pull the latest translations from Transifex run
+
+    bin/pull_translations
+
+and commit any changes.
+
```
Comment (DevneyHamilton) [inline README.md:68]: thanks for this documentation!

**Review by DevneyHamilton — APPROVED** [review]
Thanks for doing this, and for handling the issue with duplicate keys!

**Pull request merged**